app/models/concerns/smooch_language.rb

```diff
@@ -4,9 +4,10 @@ module SmoochLanguage
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def get_user_language(uid, message = {}, state = nil)
-      team = Team.find(self.config['team_id'])
-      default_language = team.default_language
+    def get_user_language(uid, message = {}, state = nil, team_id = nil)
+      team_id ||= self.config['team_id']
+      team = Team.find_by_id(team_id)
+      default_language = team&.default_language
```

Comment:
I think it's better to raise an exception here if `team` is `nil`, because really it should never be.

Reply:
Done